the-collab-lab/tcl-38-smart-shopping-list
# Issue #2: Create react routes to navigate between the “list” view and the “add an item” view

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description

This PR adds two new components, one to view the item list and one for adding new items. 

We decided to use CSS Grid to position the links at the bottom of the app because we felt that it would be more flexible as we continue to develop the app, vs something like position:fixed. We also decided to use <NavLink> rather than <Link> for routing our links because it has a built-in active class to allow custom in-line styling based on the active state.

## Related Issue
Closes Issue #2.


## Acceptance Criteria



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|  ✓ | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates


### Before
![image](https://user-images.githubusercontent.com/40015502/149225317-3a0db140-2fd8-4d98-b530-aff16c235b27.png)




### After
![image](https://user-images.githubusercontent.com/40015502/149224999-05cd2761-606a-4b49-960d-a3639ed95068.png)




## Testing Steps / QA Criteria
- git pull ck-km-create-react-routes
- run npm install
- run npm start
- click links; they should turn bold when clicked, and the URL should change in the browser
